scripts/benchmark.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
+
+import datetime
+import time
+import argparse
+import os
+import sys
+import collections
+import matplotlib.pyplot as plt
+import jinja2
```

Comment:
This comment applies to the whole PR? Are all of these python packages part of the MOOSE package or did you have to download any extra packages through mini-conda?

Reply:
They should already be in mini-conda.

Reply:
Our moose mini-conda environment that is.